src/types/wrappers/SimplifiedEnzymeConstrainedModel.jl

```diff
@@ -45,7 +45,7 @@ is retained.
 
 In the structure, the field `columns` describes the correspondence of
 stoichiometry columns to the stoichiometry and data of the internal wrapped
-model, and `total_enzyme_capacity` is the total bound on the enzyme capacity
+model, and `total_gene_product_mass_bound` is the total bound on the enzyme capacity
```

Comment:
capacity (just "enzyme consumption" would be ok here, possibly with quoted consumption)

Reply:
nah, I want the wording of the kwargs to be as similar as possible to avoid having to remember too much